minegauler/shared/utils.py

```diff
@@ -178,8 +175,8 @@ def fill(self, item):
             The item to fill the grid with.
         """
         for row in self:
-            for i in range(len(row)):
-                row[i] = item
+            for column in row:
+                column = item
```

Comment:
Your change is semantically different, and wrong - it does update the entries of `row` it just assigns values to a temporary `column` variable. Either leave as it is (revert your change) or change to something like `row[:] = (item for _ in row)`.